Documentation/operations/upgrade.rst

```diff
@@ -820,7 +832,7 @@ The cilium preflight manifest requires etcd support and can be built with:
 
 
 Example migration
-~~~~~~~~~~~~~~~~~
+$$$$$$$$$$$$$$$$$
```

Comment:
In Cilium docs we use `~` for this level of header. Please follow the same (same feedback below):

```suggestion
~~~~~~~~~~~~~~~~~
```